spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -228,6 +228,34 @@ From time to time, the container must obtain an _active context object_ for a ce
 
 If there is exactly one active instance of `Context` associated with the scope type, we say that the scope is _active_.
 
+==== Activating Built In Contexts
```

Comment:
Hmm.. I was thinking maybe putting in as the last chapter under `6. Scopes and contexts` would fit better (e.g. create `6.8`)? It just my feeling though, anywhere within contexts chapter is probably a good shot.